diff --git a/dist/cjs/index.development.js b/dist/cjs/index.development.js
index 28234248bed310531bc453013ec56b687255b22b..e7c7790fd7da60fd5433f0a00902ca128f4e27b6 100644
--- a/dist/cjs/index.development.js
+++ b/dist/cjs/index.development.js
@@ -719,6 +719,7 @@ const createHook = (reactiveState, initialState, lifeCycle, deepSelector = true,
             else {
                 traverse(re);
             }
+            return re;
         };
         const controller = new Controller(subscribeSelector, Object.is, lifeCycle, controllerList, InternalNameSpace.$$__subscribe__$$, () => cb());
         controller.run();
diff --git a/dist/cjs/index.production.js b/dist/cjs/index.production.js
index 31205f72b865416751e98c0eaff0887d8a34ee0d..ffb043580ee95bded8d4d89ae2917cd37af64156 100644
--- a/dist/cjs/index.production.js
+++ b/dist/cjs/index.production.js
@@ -421,6 +421,7 @@ const createHook = (reactiveState, initialState, lifeCycle, deepSelector = true,
             else {
                 traverse(re);
             }
+            return re;
         };
         const controller = new Controller(subscribeSelector, Object.is, lifeCycle, controllerList, InternalNameSpace.$$__subscribe__$$, () => cb());
         controller.run();
diff --git a/dist/esm/index.mjs b/dist/esm/index.mjs
index ef99ba17f77481ed9db615d0fa307bbb3f166dc8..32cade757f625e83b67858b2fef8f806be92141c 100644
--- a/dist/esm/index.mjs
+++ b/dist/esm/index.mjs
@@ -724,6 +724,7 @@ const createHook = (reactiveState, initialState, lifeCycle, deepSelector = true,
             else {
                 traverse(re);
             }
+            return re;
         };
         const controller = new Controller(subscribeSelector, Object.is, lifeCycle, controllerList, InternalNameSpace.$$__subscribe__$$, () => cb());
         controller.run();
